Ginger/GingerCoreNET/WorkSpaceLib/Solution.cs

```diff
@@ -86,7 +86,7 @@ public void SaveSolution(bool showWarning = true, eSolutionItemToSave solutionIt
                 }
                 if (solutionItemToSave != eSolutionItemToSave.ALMSettings)
                 {
-                    if (this.ALMDomain != lastSavedSolution.ALMDomain || this.ALMProject != lastSavedSolution.ALMProject || this.ALMServerURL != lastSavedSolution.ALMServerURL || this.AlmType != lastSavedSolution.AlmType)
+                    //if (this.ALMDomain != lastSavedSolution.ALMDomain || this.ALMProject != lastSavedSolution.ALMProject || this.ALMServerURL != lastSavedSolution.ALMServerURL || this.AlmType != lastSavedSolution.AlmType)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)